content/release-10-16-0/announcements-10-16-0.md

```diff
@@ -8,6 +8,60 @@ layout: bundle
 
 #### Planned
 
+##### Breaking change in the SmartREST 2.0 -- a DATE field used as a custom property will be stored as a String.
+With 10.17 release a SmartREST 2.0 DATE field used as a custom property will be stored as a String in the Cumulocity database.
+This affects REST response format.
+
+Example of previous response format (note `date` field):
+
+```json
+{
+   "source":{
+      "id":"91123"
+   },
+   "type":"TYPE-911",
+   "date":{
+      "date":{
+         "date":14,
+         "seconds":15,
+         "hours":12,
+         "month":10,
+         "year":122,
+         "timezoneOffset":-60,
+         "minutes":41,
+         "time":1668426075840,
+         "day":1
+      },
+      "offset":120
+   },
+   "c8y_TemperatureMeasurement":{
+      "T":{
+         "unit":"C",
+         "value":10
+      }
+   }
+}
+```
+
+New format:
+
+```json
+{
+   "source":{
+      "id":"91123"
+   },
+   "type":"TYPE-911",
+   "date":"2022-11-14T12:44:11.481+01:00",
```

Comment:
Maybe rename the `date` field to something like `myCustomDate` so it is not mixed with our standard time fields

Reply:
@c8y-kh4mell @martin-sag Breaking changes must be announced two release upfront, so this should be added to the 10.15 announcements as well, shouldn´t it? Still possible as they are still in preview mode.